### PR TITLE
Weth fix

### PR DESCRIPTION
**Pull request by thrilok209**

### CODE REVIEW

File: contracts/receivers/aave-v2-receiver/main.sol [hunk 1]
```diff
@@ -56,6 +56,14 @@ contract MigrateResolver is Helpers, Events {
                     TokenInterface(wmaticAddr).deposit{value: address(this).balance}();
                 }
             }
+
+            if (_token == wethAddr) {
+                TokenInterface wethPosContract = TokenInterface(wethPosAddr);
+                if (wethPosContract.balanceOf(address(this)) > 0) {
+                    WETHTokenInterface(wethAddr).deposit(address(this), abi.encode(wethPosContract.balanceOf(address(this))));
```
Comment (mubaris) [inline contracts/receivers/aave-v2-receiver/main.sol:63]: From [Matic WETH](https://explorer-mainnet.maticvigil.com/address/0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619/contracts) #L1502
`Should be callable only by ChildChainManager`

I don't think will work. ChildChainManager is [0xA6FA4fB5f76172d178d61B04b0ecd319C5d1C0aa](https://explorer-mainnet.maticvigil.com/address/0xA6FA4fB5f76172d178d61B04b0ecd319C5d1C0aa/transactions)

Comment (thrilok209) [reply contracts/receivers/aave-v2-receiver/main.sol:63]: Yess! I will remove this.

File: contracts/receivers/aave-v2-receiver/variables.sol [hunk 2]
```diff
@@ -36,12 +36,22 @@ contract Variables {
...
+     * @dev WETH POS address
+     */
+    address public constant wethPosAddr = 0xAe740d42E4ff0C5086b2b5b5d149eB2F9e1A754F;
+
+    /**
+     * @dev WETH address
+     */
+    address public constant wethAddr = 0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619;
+
+
```
Comment (thrilok209) [inline contracts/receivers/aave-v2-receiver/variables.sol:54]:
```suggestion
```

File: contracts/senders/aave-v2-migrator/main.sol [hunk 3]
```diff
@@ -88,8 +88,18 @@ contract LiquidityResolver is Helpers, Events {
         for (uint i = 0; i < _tokens.length; i++) {
             address _token = _tokens[i] == ethAddr ? wethAddr : _tokens[i];
             aave.withdraw(_token, _amts[i], address(this));
-            IERC20(_token).safeApprove(erc20Predicate, _amts[i]);
-            rootChainManager.depositFor(polygonReceiver, _token, abi.encode(_amts[i]));
+
+            if (_tokens[i] == ethAddr) {
```
Comment (KaymasJain) [inline contracts/senders/aave-v2-migrator/main.sol:92]:
```suggestion
            if (_token == wethAddr) {
```

Why not this?

**Review by KaymasJain — APPROVED** [review]

**Pull request merged**